releases/8.3/languages/en.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+return [
+    'common_header' => 'PHP 8.3 is a major update of the PHP language. Readonly amendments, Typed class constants, #[Override] attribute, Randomizer additions, performance improvements and more',
+    'documentation' => 'Doc',
+    'main_title' => 'Released!',
+    'main_subtitle' => 'PHP 8.3 is a major update of the PHP language.<br class="display-none-md">It contains many new features, including Readonly amendments, Typed class constants, Randomizer additions, performance improvements and more.',
+    'upgrade_now' => 'Upgrade to PHP 8.3 now!',
+
+    'readonly_title' => 'Readonly amendments',
```

Comment:
```suggestion
    'readonly_title' => 'Deep-cloning of readonly properties',
```
This is perhaps better, because “readonly amendments” could be anything.

Reply:
Should we also use that wording in `common_header` and `main_subtitle`, what do you think?

Reply:
Yes, that would make sense. How about something like this:

> It contains many new features, such as explicit typing of class constants, deep-cloning of readonly properties and additions to the randomness functionality. As always it also includes performance improvements, bug fixes and general cleanup.

For that “elevator pitch” I've focused on the three features that are easily understood with just a few words and no code examples and then mentioned the general improvements that go into every release.

Reply:
I like this. Concise with a slight positive tone.